src/tests/ftest/scrubber/aggregation.py

```diff
@@ -0,0 +1,56 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from scrubber_test_base import TestWithScrubber
+
+
+class TestScrubberEvictWithAggregation(TestWithScrubber):
+    # pylint: disable=too-many-nested-blocks
+    """Inject Checksum Fault with scrubber enabled
+    and scrubber threshold set to a certain value.
+    Aggregation is run on the background.
+
+    :avocado: recursive
+    """
+    def test_target_eviction_during_aggregation(self):
+        """JIRA ID: DAOS-7333
+
+        1. Start the Aggregation task.
+        2. Create checksum faults above scrubber threshold
+        and see whether SSD auto eviction works as expected.
+        :avocado: tags=all,pr,daily_regression
```

Comment:
nit: there should be a space
```suggestion
        and see whether SSD auto eviction works as expected.

        :avocado: tags=all,pr,daily_regression
```

Also, does this really need pr and daily?

Reply:
All these tests need to be moved to full_regression. Forgot to change it... We have coverage for scrubber under pr and daily regressions.

Reply:
Done.